src/table/utils/handle-accessibility.ts

```diff
@@ -28,23 +48,43 @@ export const updateFocus = ({ focusType, cell }) => {
   }
 };
 
-export const removeAndFocus = (newCoord, rootElement, setFocusedCellCoord, keyboard) => {
+export const removeAndFocus = (
+  newCoord: [number, number],
+  rootElement: Element,
+  setFocusedCellCoord: (coord: [number, number]) => void,
+  keyboard: stardust.Keyboard
+) => {
   updateFocus({ focusType: 'removeTab', cell: findCellWithTabStop(rootElement) });
   setFocusedCellCoord(newCoord);
-  keyboard.enabled && keyboard.focus();
+  keyboard.enabled && keyboard.focus?.();
 };
 
-export const handleClickToFocusBody = (cell, rootElement, setFocusedCellCoord, keyboard, totalsPosition) => {
+export const handleClickToFocusBody = (
+  cell: { rawRowIdx: number; rawColIdx: number },
+  rootElement: Element,
+  setFocusedCellCoord: (coord: [number, number]) => void,
+  keyboard: stardust.Keyboard,
+  totalsPosition: string
+) => {
   const { rawRowIdx, rawColIdx } = cell;
   const adjustedRowIdx = totalsPosition === 'top' ? rawRowIdx + 2 : rawRowIdx + 1;
   removeAndFocus([adjustedRowIdx, rawColIdx], rootElement, setFocusedCellCoord, keyboard);
 };
 
-export const handleClickToFocusHead = (columnIndex, rootElement, setFocusedCellCoord, keyboard) => {
+export const handleClickToFocusHead = (
+  columnIndex: number,
+  rootElement: Element,
+  setFocusedCellCoord: (coord: [number, number]) => void,
+  keyboard: stardust.Keyboard
+) => {
   removeAndFocus([0, columnIndex], rootElement, setFocusedCellCoord, keyboard);
 };
 
-export const handleMouseDownLabelToFocusHeadCell = (evt, rootElement, columnIndex) => {
+export const handleMouseDownLabelToFocusHeadCell = (
+  evt: BaseSyntheticEvent,
```

Comment:
This is an event from onMouseDown so I think it is a React.MouseEvent

Reply:
hmm yes that is true, for the keyboard events as well it seems like. I will change in my PR

Reply:
Looking at it again it seems like the devtools are showing the wrong type, no idea why. If you look at the type definitions for the different react types, the properties on the event match to React.MouseEvent. So set it to that

Reply:
Sure, I'll set it to MouseEvent.

Reply:
MouseEvent extends UIEvent extends SyntheticEvent extends BaseSyntheticEvent. so MouseEvent is more accurate in this case.